### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
# Math magicians: full website 🎉  ✔️  🎉  ✔️ 
- Use React Router.
- Use styles in a React app.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
-    </>
-  );
+class App extends PureComponent {
+  render() {
+    return (
+      <>
+        <Headers />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/calDesign" element={<Calculator />} />
```
Comment (t-yanick) [inline src/App.js]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` " , please kindly ensure the general layout of your calculator page matches the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) both in content and alignment. Your page is missing the "  `Let's do the math!` " text, please kindly add it and make sure it is left align.  The calculator itself occupies about 2/3 of the page and it is right aligned, please kindly fix that too.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Nizamuddin4493, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -83,15 +86,297 @@ input:active {
...
+nav {
+  width: 100%;
+}
+
+.website-top {
+  text-align: center;
+  border: 5px solid black;
+  margin-bottom: 10px;
+  background: lightgrey;
+}
```
Comment (Ugboaja-Uchechi) [inline src/index.css:121]: - [ ] [OPTIONAL] Great job following the layout 👍 Please note that this part of the wireframe doesn't need to be included in your design 🙏 

image

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @Nizamuddin4493 , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**